oneanddone/base/middleware.py

```diff
@@ -16,7 +16,8 @@ def process_request(self, request):
         if request.user.is_authenticated():
             for attempt in request.user.attempts_requiring_notification:
                 messages.warning(request,
-                                 _('The task that you were working on, "%s", has expired or become invalid '
```

Comment:
Applying this logic to all the changed files, you could get away with fewer line breaks